Question 1:
    Line 9 prints: "values added: 20"

Question 2:
    Line 13 prints: "final result : 20"

Question 3:
    Line 9 prints: "values added : 20"

Question 4:
    Line 13 returns an error because 'result' was declared with "let".Line 13 is outside of the block that it was defined in but it tries to use 'result'

Question 5:
    Line 9 will return an error because it is trying to reassign the variable "result" which was declared with const, meaning it can only be assigned once.

Question 6:
    Line 13 will return an error because it tries to use 'result' in a different block that it was declared, which is not allowed because 'result' was declared with const